src/Surfnet/StepupGateway/SecondFactorOnlyBundle/Service/Gateway/SecondFactorAdfsService.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * Copyright 2018 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupGateway\SecondFactorOnlyBundle\Service\Gateway;
+
+use Exception;
+use Psr\Log\LoggerInterface;
+use Surfnet\SamlBundle\SAML2\ReceivedAuthnRequest;
+use Surfnet\StepupGateway\GatewayBundle\Saml\ResponseContext;
+use Surfnet\StepupGateway\SecondFactorOnlyBundle\Adfs\Exception\InvalidAdfsRequestException;
+use Surfnet\StepupGateway\SecondFactorOnlyBundle\Adfs\Exception\InvalidAdfsResponseException;
+use Surfnet\StepupGateway\SecondFactorOnlyBundle\Adfs\RequestHelper;
+use Surfnet\StepupGateway\SecondFactorOnlyBundle\Adfs\ResponseHelper;
+use Symfony\Component\HttpFoundation\Request;
+
+class SecondFactorAdfsService
+{
+    /** @var RequestHelper */
+    private $adfsRequestHelper;
+
+    /** @var ResponseHelper */
+    private $adfsResponseHelper;
+
+    /**
+     * SecondFactorAdfsService constructor.
```

Comment:
The comment for this constructor is not that usefull, All of the parameters are type hinted so additional phpdoc type definition is not required here. Rule of thumb is: only add the phpdoc param annotations if they actually add value, for example when primitive types are in the methods signature.

Please fix this for other newly added classes too!

Reply:
Removed